tests/v2/validation/Jenkinsfile.e2e

```
@@ -175,8 +177,10 @@ node {
               sh "docker rm -v ${buildTestContainer}"
               sh "docker stop ${golangTestContainer}"
               sh "docker rm -v ${golangTestContainer}"
-              sh "docker stop ${cleanupTestContainer}"
-              sh "docker rm -v ${cleanupTestContainer}"
+              if ("${env.RANCHER_CLEANUP}" == "True" || "${env.RANCHER_CLEANUP}" == "true") {
```

After giving this part some thought, I don't have a strong opinion on this cause either way it's gonna work, but what do you all think about this:

`def cleanup = env.RANCHER_CLEANUP.toLowerCase()` // somewhere at the above while assigning env vars

```suggestion
              if (cleanup.toBoolean()) {
```

---

I don't see that pipeline file being used in any job but I'll give it a shot